types/fh-sync.d.ts

```diff
@@ -1,48 +1,97 @@
 declare module 'fh-sync' {
 
-  interface PendingWorkerBackoff {
-    strategy: string;
-    max: string;
-  }
-
-  interface AckWorkerBackoff {
-    strategy: string;
-    max: string;
-  }
-
-  interface SyncWorkerBackoff {
-    strategy: string;
-    max: number;
-  }
-
+  /**
+   * Options used to initialize Sync Server
+   */
   interface SyncGlobalOptions {
```

Comment:
I think the docs here should be a `@typedef` with each of the keys here as a `@property`. We can then specify their default values using the JSDoc syntax. This will make the generated docs look better I think.

```
@typedef {Object} SyncGlobalOptions Options for configuring the sync server using setConfig
@property {Number} [datasetClientUpdateConcurrency=10]
...
```

WDYT?